ci_config.json

```diff
@@ -18,7 +18,8 @@
     "libxml2", "libxmlpp", "lua", "lzo2", "mocklibc", "mpdecimal", "netstring-c",
     "nng", "openh264", "openssl", "pcre2", "protobuf", "protobuf-c", "pugixml",
     "quazip", "rdkafka", "reflex", "sdl2", "sdl2_image", "sdl2_mixer",
-    "sdl2_net", "sdl2_ttf", "termbox", "unit-system", "xxhash", "zstd"
+    "sdl2_net", "sdl2_ttf", "termbox", "unit-system", "wayland-protocols",
```

Comment:
This is intended to mark packages as "auto-fail CI, and in exchange mark as known-broken for the manually triggered build_all.yml".

For software that fails on Windows, but is intended to since it is linux-specific, see how liburing works later in this file.